From d812389b4556142a5ea9dbcec3f681c3b694ba09 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:49:58 -0400
Subject: [fs] nfsd: nfsd4_replay_cache_entry should be static

Message-id: <1403560314-32525-57-git-send-email-bfields@redhat.com>
Patchwork-id: 84478
O-Subject: [Patch RHEL7 056/172] nfsd4: nfsd4_replay_cache_entry should be static
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

This isn't actually used anywhere else.

Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: 3ca2eb981435d5350c395d5266fac484b5cb1af9
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/nfs4state.c b/fs/nfsd/nfs4state.c
index a07d98c..a08b58d 100644
--- a/fs/nfsd/nfs4state.c
+++ b/fs/nfsd/nfs4state.c
@@ -1596,7 +1596,7 @@ nfsd4_enc_sequence_replay(struct nfsd4_compoundargs *args,
  * The sequence operation is not cached because we can use the slot and
  * session values.
  */
-__be32
+static __be32
 nfsd4_replay_cache_entry(struct nfsd4_compoundres *resp,
     struct nfsd4_sequence *seq)
 {
diff --git a/fs/nfsd/xdr4.h b/fs/nfsd/xdr4.h
index d278a0d..5ea7df3 100644
--- a/fs/nfsd/xdr4.h
+++ b/fs/nfsd/xdr4.h
@@ -574,8 +574,6 @@ extern __be32 nfsd4_setclientid_confirm(struct svc_rqst *rqstp,
   struct nfsd4_compound_state *,
   struct nfsd4_setclientid_confirm *setclientid_confirm);
 extern void nfsd4_store_cache_entry(struct nfsd4_compoundres *resp);
-extern __be32 nfsd4_replay_cache_entry(struct nfsd4_compoundres *resp,
-  struct nfsd4_sequence *seq);
 extern __be32 nfsd4_exchange_id(struct svc_rqst *rqstp,
   struct nfsd4_compound_state *, struct nfsd4_exchange_id *);
 extern __be32 nfsd4_backchannel_ctl(struct svc_rqst *, struct nfsd4_compound_state *, struct nfsd4_backchannel_ctl *);
-- 
1.7.1